README_BEAGLEBONE_BLACK.md

```diff
@@ -0,0 +1,162 @@
+
+    ssh debian@beaglebone # default password is temppwd
+    sudo passwd debian # change password
+    sudo /opt/scripts/tools/grow_partition.sh # expand file system. skip if you are running on the emmc
+    sudo reboot # and log in again with ssh
+
+    # log in again with ssh
+    ssh debian@beaglebone
+    sudo apt-get update
+    sudo apt-get upgrade
+    sudo apt-get dist-upgrade
+    sudo apt-get install libsamplerate0-dev libsndfile1-dev libasound2-dev libavahi-client-dev libreadline-dev \
+        libfftw3-dev libudev-dev cmake git build-essential python-dev alsa-utils cpufrequtils
+
+### Step 3: Compile, install and configure jackd (no d-bus)
+
+    cd ~ # or cd into the directory where you'd like to build jack2
+    git clone git://github.com/jackaudio/jack2 --depth 1
+    cd jack2
+    ./waf configure --alsa --libdir=/usr/lib/arm-linux-gnueabihf/
+    ./waf build
+    sudo ./waf install
+    sudo ldconfig
+    cd ..
+    rm -rf jack2
+    sudo sh -c "echo @audio - memlock 256000 >> /etc/security/limits.conf"
+    sudo sh -c "echo @audio - rtprio 75 >> /etc/security/limits.conf"
+
+    # `-dhw:0` is the internal soundcard. Use `-dhw:1` for USB soundcards. `aplay -l` will list available devices.
+    # Use `nano ~/.jackdrc` to edit jack settings.
+    echo /usr/local/bin/jackd -P75 -p16 -dalsa -dhw:0 -r44100 -p1024 -n3 > ~/.jackdrc
+
+    exit # and ssh in again to make the limits.conf settings work
+
+### Step 4: Compile and install SuperCollider
+
+    cd ~ # or cd into the directory where you'd like to clone and build supercollider
+    git clone --recurse-submodules https://github.com/supercollider/supercollider.git
+    cd supercollider
+    mkdir build && cd build
+    cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DSUPERNOVA=OFF -DNATIVE=ON -DSC_IDE=OFF -DNO_X11=ON \
+        -DSC_QT=OFF -DSC_ED=OFF -DSC_EL=OFF -DSC_VIM=ON ..
+    cmake --build . --config Release --target all
+    sudo cmake --build . --config Release --target install
+    sudo ldconfig
+
+
+Usage
+-----
+
+To use SuperCollider, just open a terminal and execute `sclang`.
+
+**Done!** See below for other usage notes and tips.
+
+sc3-plugins
+-----------
+
+To compile and install sc3-plugins, follow the instructions in the [sc3-plugins README](https://github.com/supercollider/sc3-plugins).
+
+Autostart
+---------
+
+To automatically run SuperCollider code at system boot:
+
+    cat >~/autostart.sh <<EOF
+    #!/bin/bash
+    export PATH=/usr/local/bin:$PATH
+    export DISPLAY=:0.0
+    sleep 10 # can be lower (5) for rpi3
+    sclang mycode.scd
+    EOF
+
+    chmod +x ~/autostart.sh
+    crontab -e # and add the following line to the end:
+        @reboot cd /home/pi && ./autostart.sh
+
+    nano ~/mycode.scd # And add your code inside a waitForBoot. For example:
+        s.waitForBoot{ {SinOsc.ar([400, 404])}.play }
+
+    sudo reboot # the sound should start after a few seconds
+
+Login with ssh and run `killall jackd sclang scsynth` to stop the sound.
```

Comment:
killall isn't installed on the most barebone bbb image (console).  not sure about the iot or lxqt.  so safer to use...
```
pkill jackd; pkill sclang; pkill scsynth
```